client/src/components/QuestionsAndAnswers/AddModal.jsx

```diff
@@ -8,9 +8,11 @@ const AddModal = ({submitClickHandler, isQuestion}) => {
   return (
     <div>
       <div>AddModal</div>
-      <button onClick={() => submitClickHandler()}>Add a ___ +</button>
+      {isQuestion ? <button onClick={() => submitClickHandler()}>Add a Question +</button> : <button onClick={() => submitClickHandler()} style={{padding: 0, background: "none", border: "none", color: "blue", textDecoration: "underline", textTransform: "none" }}>Add Answer</button>}
```

Comment:
I would avoid defining a new anonymous function when unnecessary. For example, in the button onClick handler it looks like you could just pass `onClick={submitClickHandler}`.

Also it may be helpful to use more lines to improve readability in the future, especially for ternaries: 
```
{isQuestion ? (
  <button>Add a Question +</button>
) : (
  <button
    onClick...
    style...
  >
     Add answer
  </button>
)}
```

Reply:
did not know you could just do that, will do going forward, will fix